### PR TITLE
Rtn2g

### PR DESCRIPTION
**Pull request by EvgenyKarkan**

### CODE REVIEW

File: Spec/RealtimeClientConnection.swift [hunk 1]
```diff
@@ -170,6 +170,41 @@ class RealtimeClientConnection: QuickSpec {
...
+                        let stateChange = stateChange!
+                        let state = stateChange.current
+                        let errorInfo = stateChange.reason
+                        switch state {
+                        case .Failed:
+                            AblyTests.checkError(errorInfo, withAlternative: "Failed state")
+                            done()
+                        case .Connected:
+                            if let transport = client.transport as? TestProxyTransport, let query = transport.lastUrl?.query {
+                                expect(query).to(haveParam("lib", withValue: ARTDefault.libraryVersion()))
```
Comment (mattheworiordan) [inline Spec/RealtimeClientConnection.swift]: As mentioned in the REST test, I would be a lot happier know that the `lib` actually matches a string or regex we can read in the tests.

**Comment by mattheworiordan** [comment]
Thanks @EvgenyKarkan, looks good, only small comment re: test readability and predictability.  Out of interest, why did you commit ed233af?

**Comment by EvgenyKarkan** [comment]
Thanks for feedback @mattheworiordan 
As per your question, the short answer is - because of `Ably` code style reason.
After working on test on dependant branch `RTN2g`, I did a such fix on dependency branch `RSC7b` and then decided to merge it.

**Comment by mattheworiordan** [comment]
Ok @EvgenyKarkan. This is probably exceptional because normally one PR (branch) can be based on another, but you need to be a collaborator in Github to do that.  If we continue further, we'll of course make you a collaborator

File: Spec/RealtimeClientConnection.swift [hunk 2]
```diff
@@ -170,6 +170,41 @@ class RealtimeClientConnection: QuickSpec {
...
+                        let stateChange = stateChange!
+                        let state = stateChange.current
+                        let errorInfo = stateChange.reason
+                        switch state {
+                        case .Failed:
+                            AblyTests.checkError(errorInfo, withAlternative: "Failed state")
+                            done()
+                        case .Connected:
+                            if let transport = client.transport as? TestProxyTransport, let query = transport.lastUrl?.query {
+                                expect(query).to(haveParam("lib", withValue: "ios-0.8.0-beta.1"))
```
Comment (tcard) [inline Spec/RealtimeClientConnection.swift]: Actually [we're at 0.8.4 now](https://github.com/ably/ably-ios/releases/tag/0.8.4). I didn't even know that there was such thing as a bundle version. Can you update it?

**Comment by EvgenyKarkan** [comment]
Hi @tcard , should I also update `Ably` library bundle version inside `Info.plist` file?

**Comment by tcard** [comment]
@EvgenyKarkan Yes, please.

**Comment by mattheworiordan** [comment]
@tcard you happy for this to be merged?

**Comment by tcard** [comment]
Yes, LGTM.

**Pull request merged**